pkg/minikube/bootstrapper/images/repo.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 // OldDefaultKubernetesRepo is the old default Kubernetes repository
-const OldDefaultKubernetesRepo = "k8s.gcr.io"
+const OldDefaultKubernetesRepo = "registry.k8s.io"
```

Comment:
Changing the old registry to be the same as the new, makes this rather meaningless.

Reply:
ah yeah - sorry, this was a more generic find & replace. Will update